PR: https://github.com/odoo/odoo/pull/

From: 7978d4ba0f093c4b5400dbb1776ff5c0416e203b
From: Yannick Tivisse
Date: 2018-09-10 12:23:42

Breaking data model changes score: 4, change matches:
-    _inherit = 'res.users'
-    has_group_warning_account = fields.Boolean(
-    has_group_cash_rounding = fields.Boolean(
-    group_account_user = fields.Selection(

Total Changes: 22

Revert "[IMP] *: Define groups on res.users models"

This reverts commit 99f497b390e680629dce3a7e2e282ada74ee9884.

================================= pseudo patch: =================================

--- a/addons/account/models/__init__.py
+++ b/addons/account/models/__init__.py
@@ -15,4 +15,3 @@ from . import company
 from . import res_config_settings
 from . import account_cash_rounding
 from . import reconciliation_widget
-from . import res_users

--- a/addons/account/models/res_users.py
+++ b/None
@@ -1,21 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-
-class ResUsers(models.Model):
-    _inherit = 'res.users'
-
-    has_group_warning_account = fields.Boolean(
-        'A warning can be set on a partner (Account)', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='account.group_warning_account')
-
-    has_group_cash_rounding = fields.Boolean(
-        'Allow the cash rounding management', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        group_xml_id='account.group_cash_rounding')
-
-    group_account_user = fields.Selection(
-        selection=lambda self: self._get_group_selection('base.module_category_accounting_and_finance'),
-        string='Accounting & Finance', compute='_compute_groups_id', inverse='_inverse_groups_id',
-        category_xml_id='base.module_category_accounting_and_finance')
